spec/ldp/rdf_source_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe Krikri::LDP::RdfSource do
```

Comment:
This file should be removed. These tests are handled in shared examples, here: https://github.com/dpla/KriKri/pull/60/files#diff-dadee38ed9146031364c384d8615bd03